quarkus.ls.ext/com.redhat.quarkus.ls/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.0/apache-maven-3.6.0-bin.zip
```

Comment:
that's an old version of Maven, can you update the Maven wrappers everywhere as part of this PR? Just call mvn -N io.takari:maven:0.7.7:wrapper at the root of each project containing the wrapper